### DISCUSSION TITLE
GPU acceleration?

### DISCUSSION THREAD

**Discussion by ramBrain** [simnibs/simnibs#206] [4. Ideas]
Hi all,

As per title, I wonder if it is feasible to implement `charm` to be able to use the GPU when available? I presume that would significantly enhance the performance.

Thank you,
Ramtin

**Comment by oulap** [comment]
At the moment that would require quite a bit of work at least for the segmentation algorithm, which does not directly support GPU acceleration. But speeding up the charm pipeline is definitely on the list of things that we plan to work on.
